NEWS.md

```diff
@@ -1,4 +1,24 @@
-## 2.1.0 - Unreleased
+## 2.2.0 - Unreleased
+
+## 2.1.0 - Released (Poppy R2 2023)
+
+The focus of this release was to update dependencies and remove unnecessary code
+
+[Full Changelog](https://github.com/folio-org/mod-orders/compare/v2.0.0...v2.1.0)
+
+### Stories
+
+* [MODEBSNET-65](https://issues.folio.org/browse/MODEBSNET-65) spring boot upgrade
+* [MODEBSNET-64](https://issues.folio.org/browse/MODEBSNET-64) Folio Spring base updated
+* [MODEBSNET-62](https://issues.folio.org/browse/MODEBSNET-62) Delete unused Kafka and database references
+* [FOLIO-3678](https://issues.folio.org/browse/FOLIO-3678) Use GitHub Workflows api-lint and api-schema-lint and api-doc
+* [MODEBSNET-59](https://issues.folio.org/browse/MODEBSNET-59) Migrate to folio-spring-support v7.0.0
+* [MODEBSNET-58](https://issues.folio.org/browse/MODEBSNET-58) Delete beginFolioExecutionContext from TestBase
+
+### Dependencies
```

Comment:
Is it new requirement or agreement?

Reply:
not requirement, looks good info, I read from documentation 
https://wiki.folio.org/pages/viewpage.action?pageId=96404780 
![image](https://github.com/folio-org/mod-ebsconet/assets/113523904/4c8aacdd-7574-473c-8c95-8784555af023)
this info once help me to fix one issue.

Reply:
thanks, lgtm